### PR TITLE
Quote message and UI

### PR DESCRIPTION
**Pull request by akanksha-coditas**
1. Added quote message feature with UI and implementation.
2. Changed colors as per the requirement.
3. Refactored display picture code.
4. Added columns for quoted message and respective fields in db.

### CODE REVIEW

File: lib/features/chat_module/presentation/pages/group_chat/widgets/navigation_drawer.dart
Comment (SumitAmbhoreCoditas) [inline lib/features/chat_module/presentation/pages/group_chat/widgets/navigation_drawer.dart:1]: remove dead code and styles

Comment (SumitAmbhoreCoditas) [inline lib/features/chat_module/presentation/pages/group_chat/widgets/navigation_drawer.dart:1]: use test styles from theme

File: lib/features/components/single_message.dart
Comment (SumitAmbhoreCoditas) [inline lib/features/components/single_message.dart:1]: avoid using copy with and use mostly .withThin . withBold extensions

File: lib/features/chat_module/domain/entities/group_entity.dart
[the remark this answers is not in the document]
Comment (akanksha-coditas) [reply lib/features/chat_module/domain/entities/group_entity.dart:1]: done

**Pull request merged**